### PR TITLE
More useful usage message for `rocq`

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: topbin/rocq.ml [hunk 1]
```diff
@@ -29,6 +26,79 @@ let with_sibling_exe opts prog args =
...
+  ("dep", "Print dependencies for compiling Rocq files", Dep);
+  ("doc", "Generate documentation from a Rocq source file", Doc);
+  ("wc", "Count lines of code in a Rocq source file", Wc);
+  ("workmgr", "Control the number of parallel workers used by Rocq", Workmgr);
+  ("tex", "Process Rocq code in a Latex document", Tex);
+  ("makefile", "Generate a Makefile to compile a Rocq project", Makefile);
+  ("timelog2html", "Combine timing information and a Rocq source file", Timelog2Html);
+]
+
+let print_usage fmt () =
```
Comment (gares) [inline topbin/rocq.ml:66]: You may want to have a look at the `Format.pp_open_tbox` family of functions to do the tabulation and doc wrapping.

The table in https://github.com/LPCIC/elpi/blob/master/src/parser/README.md is printed by parser_config.ml using that kind of boxes.

Comment (SkySkimmer) [reply topbin/rocq.ml:66]: it works now, no need to get fancy

**Review by proux01 — APPROVED** [review]

**Comment by proux01** [comment]
Note: when doing `rocq compile --help` as advised I get `Usage: coqc ...`. Apparently, we have some more renaming left (in a further PR).

**Comment by proux01** [comment]
@coqbot merge now

**Pull request merged**